src/firefox/ffBrowser.ts

```diff
@@ -139,14 +141,21 @@ export class FFBrowser extends EventEmitter implements Browser {
     });
   }
 
-  async close() {
+  async _disconnect() {
     await Promise.all(this.contexts().map(context => context.close()));
     helper.removeEventListeners(this._eventListeners);
     const disconnected = new Promise(f => this.once(Events.Browser.Disconnected, f));
     this._connection.close();
     await disconnected;
   }
 
+  async close() {
+    if (this._closeOverride)
```

Comment:
nit: Pavel suggested you could use `this.__server__.close()` if __server__ is set instead of passing the callback. *Broser.close() implementation would be a bit more obvious then.

Reply:
We can make it this._server to make it more legit

Reply:
done as `_ownedServer`